src/embedded.ts

```diff
@@ -241,6 +256,18 @@ export class RoomWidgetClient extends MatrixClient {
         return {};
     }
 
+    public async getOpenIdToken(): Promise<IOpenIDToken> {
+        const token = await this.widgetApi.requestOpenIDConnectToken();
+        // the IOpenIDCredentials from the widget-api and IOpenIDToken form the matrix-js-sdk are compatible.
+        // we still recreate the token to make this transparent and catch'able by the linter in case the types change in the future.
+        return <IOpenIDToken>{
+            access_token: token.access_token,
+            expires_in: token.expires_in,
+            matrix_server_name: token.matrix_server_name,
+            token_type: token.token_type,
+        };
+    }
+
```

Comment:
Was this change and the rest of the widget API (content loaded thing) changes intentional within the scope of this RP? 🙂

Reply:
Oh, this will have been ported in from the livekit branch. It probably should have gone via develop first ideally rather than be part of this PR.